docs/index.md

```diff
@@ -1,19 +1,25 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: PowerCards
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
```

Comment:
can remove this since we not using codecov and no need for it yet.

Reply:
Oh we for real gonna discard it forever? I didnt know it was optional